database/index.js

```diff
@@ -0,0 +1,158 @@
+const db = require('./db.js');
+
+const getProducts = (page, count) => {
+  let queryString = 'SELECT * FROM products ORDER BY id OFFSET $1 LIMIT $2';
+  let offset = (page - 1) * count;
+  let params = [offset, count];
+  return db.pool
+    .connect()
+    .then(client => {
+      return client
+      .query(queryString, params)
+      .then(results => {
```

Comment:
Instead of nesting it like this, you can just return `client.query(queryString, params)` and then close up the then block, starting the new then in a new block. Helps with semantics.

Reply:
Also stops the nesting issue